src/gui/transferlistdelegate.cpp

```diff
@@ -231,7 +236,7 @@ QSize TransferListDelegate::sizeHint(const QStyleOptionViewItem & option, const
     return size;
 }
 
-QString TransferListDelegate::getStatusString(const int state) const
+QString TransferListDelegate::getStatusString(const int state)
```

Comment:
Not really but it was already like that and I wouldn't want to break some coding style rules here so kept it.